.github/workflows/ci-base-tests.yml

```diff
@@ -1,6 +1,9 @@
 name: SMARTS CI Base Tests
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - disabled
```

Comment:
this would disable the tests right? We probably want to fix it instead of disabling it

Reply:
Yes, sorry I forgot to enable those tests after just testing the ultra-ci by itself. I have restored the others tests as well